Makefile

```diff
@@ -53,6 +53,9 @@ json-check: ## Check all JSONs for basic syntax
 	@echo "Run JSON checker"
 	python3 utils/json_check.py
 
+openapi-check:
```

Comment:
Do we still have openapi-check in `make before_commit`?

Reply:
@Sergey1011010 I removed openapi check from the integration tests. It runs docker which obviously cannot be used easily from another container.

Reply:
I mean that `make before_commit` should run locally the same checkers as are in CI. Not a big deal, we can fix it later.

Reply:
I agree, but using containers limits us which and how we can run the checks.

Reply:
@Sergey1011010 I added it to `make before_commit`